packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -22,8 +22,11 @@ import { useAdapter } from './adapter/ChatAdapterProvider';
 import { ChatCompositeOptions } from './ChatComposite';
 import { ChatHeader, getHeaderProps } from './ChatHeader';
 import { FileUploadButtonWrapper as FileUploadButton } from './file-sharing/FileUploadButton';
+import { FileCard } from './file-sharing/FileCard';
```

Comment:
Import all these from `file-sharing` module directly.
You might need to conditionally remove them